---
title: L’opérande 'Throw' doit dériver de 'System.Exception'
ms.date: 07/20/2015
f1_keywords:
- vbc30665
- bc30665
helpviewer_keywords:
- BC30665
ms.assetid: 7c228087-39ea-4b30-a410-6ba711e67e5e
ms.openlocfilehash: e9e43605d5b3a752e4024414eeb87dea60d118b3
ms.sourcegitcommit: bf5c5850654187705bc94cc40ebfb62fe346ab02
ms.translationtype: MT
ms.contentlocale: fr-FR
ms.lasthandoff: 09/23/2020
ms.locfileid: "91068906"
---
# <a name="throw-operand-must-derive-from-systemexception"></a>L’opérande 'Throw' doit dériver de 'System.Exception'

L’argument fourni à `Throw` doit être une instance de `System.Exception` ou une instance d’une classe dérivée de `System.Exception`.  
  
 **ID d’erreur :** BC30665  
  
## <a name="to-correct-this-error"></a>Pour corriger cette erreur  
  
- Utilisez un argument dérivé de `System.Exception`, comme dans l’exemple suivant.  
  
    ```vb
    Throw New System.Exception("This is an error.")  
    ```  
  
## <a name="see-also"></a>Voir aussi

- [Throw, instruction](../language-reference/statements/throw-statement.md)
- [Try...Catch...Finally (instruction)](../language-reference/statements/try-catch-finally-statement.md)
- <xref:System.Exception?displayProperty=nameWithType>
- [Gestion et levée d’exceptions dans .NET](../../standard/exceptions/index.md)
